.github/workflows/build-preview-environment.yml

```
@@ -0,0 +1,108 @@
+      env_name: ${{ steps.env-name.outputs.PR_ENV_NAME }}
+    steps:
+      - name: Checkout the Tool and actions
+        uses: actions/checkout@v4
+        with:
+          ref: ${{ env.REF }}
+          fetch-depth: 1
+
+      - name: "Sanitize ENV name"
+        id: sanitize_env
+        shell: bash
+        run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:4:53: Double quote to prevent globbing and word splitting

(shellcheck)